### ISSUE TITLE
documentation suggestions

### ISSUE THREAD

**Issue by PhilipOakley** [krobelus/git-branchstack#2]
Hi, came across from the git user forum announcement (maybe also send to the git@vger.kernel.org list as well?). 

Just a few tidbits of clarification I noticed when trying to understand what was being said (or could be misunderstood) in the Readme about "topic tags".

Can the phrase "topic tags" be used consistently, from the beginning? I wasn't sure what it was referring to when it finally arrived in the text (it seems a reasonable name)

In the intro "commit messages starting with `[<topic>]` where `<topic>` is a valid branch name" it is easy to miss the two points:

* that the topic tag's name is enclosed in square brackets `[...]`, and
* that the topic tag name isn't (a rule?) an existing branch-name - rather the name can't use characters that aren't allowed in branch names (i.e. possible different views about what a `valid branch` means)

**Comment by krobelus** [comment]
> Hi, came across from the git user forum announcement
> maybe also send to the git@vger.kernel.org list as well? 

Right. I wasn't sure if that's appropriate.  Though it would be nice if I
could use Git commands to create branches, instead of the Python implementation
from git-revise.  I'll try formulate my use case ("git merge-tree" might
work but I also want to keep gi-revises's interactive way of resolving
conflicts without touching my worktree).
It looks like Git's new merge-ort backend is designed to perform more merges
in-memory (a bit like git-revise), that sounds promising.

> Just a few tidbits of clarification I noticed when trying to understand what was being said (or could be misunderstood) in the Readme about "topic tags".

Thanks for sharing, very much appreciated.

> In the intro "commit messages starting with `[<topic>]` where `<topic>` is a valid branch name" it is easy to miss the two points:
> * that the topic tag's name is enclosed in square brackets `[...]`, and

yeah, the [ ] might even look as if it's optional on the first glance..

> * that the topic tag name isn't (a rule?) an existing branch-name

Of course :facepalm:
